src/main/java/appeng/core/worlddata/DimensionData.java

```diff
@@ -153,19 +153,19 @@ public void setStoredSize( int dim, int targetX, int targetY, int targetZ )
 	@Override
 	public void sendToPlayer( @Nullable final NetworkManager manager )
 	{
-		if( manager != null )
-		{
-			for( int newDim : this.config.get( PACKAGE_DEST_CATEGORY, PACKAGE_KEY_CATEGORY, PACKAGE_DEF_CATEGORY ).getIntList() )
-			{
-				manager.scheduleOutboundPacket( ( new PacketNewStorageDimension( newDim ) ).getProxy() );
-			}
-		}
-		else
-		{
+//		if( manager != null )
+//		{
+//			for( int newDim : this.config.get( PACKAGE_DEST_CATEGORY, PACKAGE_KEY_CATEGORY, PACKAGE_DEF_CATEGORY ).getIntList() )
+//			{
+//				manager.scheduleOutboundPacket( ( new PacketNewStorageDimension( newDim ) ).getProxy() );
+//			}
+//		}
+//		else
+//		{
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)